e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/select-embed-options.cy.spec.ts

```diff
@@ -273,6 +273,24 @@ describe(suiteTitle, () => {
       cy.findByRole("heading", { name: "Email this dashboard" }).should(
         "be.visible",
       );
+
+      /**
+       * It seems `should("be.visible")` above doesn't not work in iframe.
+       * It can only check the existence of the element but not its visibility.
+       */
```

Comment:
I just realized we already have `should("be.visible")` above, but it doesn't seem to work in iframe. So I resort to checking the rough dashboard width instead.